packages/documentation/src/layout/inline/inline.mdx

```diff
@@ -0,0 +1,22 @@
+import { Canvas, Meta } from "@storybook/addon-docs/blocks";
+import * as stories from "./inline.stories.ts";
+
+<Meta of={stories} />
+
+# Inline
+
+Layout content horizontally with adjustable spacing, alignment, and positioning.
+
+<Canvas of={stories.Default} />
+
+## Wrap
+
+<Canvas of={stories.Wrap} />
+
+## Alignment
+
+<Canvas of={stories.Alignment} />
+
+## Positioning
+
+<Canvas of={stories.Positioning} />
```

Comment:
Die Klassen und die flex-property heißen beide `justify`. Dann sollten wir das in der Story auch beibehalten.